pyproject.toml

```diff
@@ -5,6 +5,7 @@ requires = ["setuptools", "wheel"]
 # build-backend = "setuptools.build_meta"
 
 [tool.black]
+target-version = ["py38", "py39", "py310"]
```

Comment:
This change actually resulted in some additional formatting changes that were apparently not happening if we also targeted `python 3.7`